lib/team_api/team_member_schema.json

```diff
@@ -0,0 +1,84 @@
+{
+  "team": {
+    "title": "team - [team.member].yml files",
+    "description": "Team member metadata",
+    "$schema": "http://json-schema.org/draft-04/hyper-schema",
+    "type": "object",
+    "properties": {
+      "name": {
```

Comment:
I don't want to both add a schema *and* change the standards all at the same time. I agree that ID might be more descriptive of what the field is used for, but that should be a separate issue.